src/librustc/ty/fold.rs

```diff
@@ -260,8 +260,8 @@ impl<'a, 'gcx, 'tcx> TyCtxt<'a, 'gcx, 'tcx> {
             /// ^          ^          ^     ^
             /// |          |          |     | here, would be shifted in 1
             /// |          |          | here, would be shifted in 2
-            /// |          | here, would be INNTERMOST shifted in by 1
-            /// | here, initially, binder would be INNERMOST
+            /// |          | here, would be innermost shifted in by 1
+            /// | here, initially, binder would be innermost
```

Comment:
can prob change these back to CAPITALS, but it looks like the original had a typo (`INNTERMOST`) though, we should fix that :)